src/main/scala/mesosphere/marathon/MarathonModule.scala

```diff
@@ -140,10 +141,9 @@ class MarathonModule(conf: MarathonConf, http: HttpConf, zk: ZooKeeperClient)
   @Singleton
   def provideStore(): PersistentStore = {
     def directZK(): PersistentStore = {
-      implicit val timer = com.twitter.util.Timer.Nil
       import com.twitter.util.TimeConversions._
-      val sessionTimeout = conf.zooKeeperSessionTimeout.get.map(_.millis).getOrElse(30.minutes)
-      val connector = NativeConnector(conf.zkHosts, None, sessionTimeout, timer)
+      val sessionTimeout = conf.zooKeeperSessionTimeout().millis
+      val connector = NativeConnector(conf.zkHosts, None, sessionTimeout, new JavaTimer(true))
```

Comment:
Please include the name of the parameter when creating the JavaTimer (isDaemon).